example/admin.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:firedart/firedart.dart';
+
+/// Demonstrates the Firestore Admin API using a privileged service account.
+/// Operations bypass all Firestore database rules that apply to regular users.
+///
+/// Usage:
+/// Create a service account with permission to access Firestore.
+/// See the console:
+/// https://console.firebase.google.com/u/0/project/_/settings/serviceaccounts/adminsdk
+/// Download and save the service account json file.
+/// ****** DO NOT CHECK THIS FILE INTO git! *****
+/// It is highly privileged and should be kept secret.
+///
+/// If you want to pass the file via environment variables:
+///   Set the environment variable GOOGLE_APPLICATION_CREDENTIALS to the
+///   path to this file.  For example:
+///   GOOGLE_APPLICATION_CREDENTIALS=./service-account.json
+///   Then you can use ServiceAccount.fromEnvironmentVariable();
+///   You may specify your own environment variable key optionally if you'd like.
+///
+/// If you want to pass the file via direct json string:
+///   Copy the service account json and use ServiceAccount.fromJson(String);
+///   Again, DO NOT PUSH THE FILE HOLDING THIS DATA TO A PUBLIC REPOSITORY.
```

Comment:
This still seems too long and perhaps unnecessary since a developer implementing a service account probably has some idea about what they're doing.
I'm also not a fan of documentation "screaming" in all caps or otherwise admonishing users for things they may already know.

How about simply adding the three different ways below with two of them commented out?
I believe the API is descriptive enough that it doesn't need to be documented further.

``` dart
Future main() async {
  // Note: Beware of pushing your credentials to source control.
  var serviceAccount = ServiceAccount.fromJson('Content of your json file');
//  var serviceAccount = ServiceAccount.fromEnvironmentVariable();
￼  FirebaseAuth.initialize(apiKey, VolatileStore(), serviceAccount: serviceAccount));
```